okhttp/src/main/java/io/grpc/okhttp/Headers.java

```diff
@@ -45,8 +46,8 @@
    * creating a stream. Since this serializes the headers, this method should be called in the
    * application thread context.
    */
-  public static List<Header> createRequestHeaders(Metadata headers, String defaultPath,
-      String authority, String userAgent) {
+  public static List<Header> createRequestHeaders(
```

Comment:
This is a public API. I'm not sure it's appropriate to delete the old signature. cc @ejona86 . Even decided to delete it, should keep the old signature for at least one release cycle and mark it with deprecated. Better to mark the new one with `@ExperimentalApi`.